spec/features/work_packages/new_work_package_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'features/work_packages/details/inplace_editor/work_package_field'
+require 'features/work_packages/work_packages_page'
+
+describe 'new work package', js: true, retry: 3, retry_wait: 5 do
```

Comment:
ahh, I am sorry. It is in my headless testing branch I am working on. https://github.com/basiszwo/openproject/blob/headless-testing/spec/support/rspec_retry.rb#L38